tests/ui/pages/data_entity/data_entity.page.ts

```diff
@@ -0,0 +1,57 @@
+import BasePage from '../base-page';
+
+const SELECTORS = {
+  addOwner: `text=Add Owner`,
+  inputName: `[placeholder="Search name"]`,
+  inputRole: `[placeholder="Search title"]`,
+  autocomplete: text => `[role="presentation"]:has-text('${text}')`,
+  addOwnerButton: `[type="submit"]`,
+  addTag: `.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeSmall:has-text('Add Tags')`,
+  inputTagName: `[role="combobox"]`,
+  saveButton: `[type="submit"]`,
+  ownerString: text => `.sc-dxtBLK.hWlgkE:has-text('${text}')`,
+  tagString: text => `.MuiTypography-root.MuiTypography-body1.sc-jdAMXn:has-text('${text}')`,
+    tag: text => `p:has-text('${text}')`,
+    addGroup: `.MuiButtonBase-root.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeSmall:has-text('Add to group')`
+};
+
+export default class DataEntityPage extends BasePage {
```

Comment:
My suggestion is to create proper files for all DataEntitty page-related tabs i.e. Overvie, Structure, Linege, History, Alerts, Actyvity and inherit the DataEntityPage from them. In this file store only the top menu with these tabs.
For now all methods are related not to the DataEntityPage but to Overvie